string base_id
string name
string prefecture
string city
float32 latitude
float32 longtitude

float32 temperature
float32 precipitation
float32 uv_index
